internal/bminventory/inventory.go

```diff
@@ -854,8 +854,8 @@ func (b *bareMetalInventory) DownloadISOInternal(ctx context.Context, infraEnvID
 	exists, err := b.objectHandler.DoesObjectExist(ctx, imgName)
 	if err != nil {
 		log.WithError(err).Errorf("Failed to get ISO for cluster %s", infraEnv.ID.String())
-		b.eventsHandler.AddEvent(ctx, infraEnvID, nil, models.EventSeverityError,
-			"Failed to download image: error fetching from storage backend", time.Now())
+		eventgen.SendDownloadImageFailedFetchEvent(ctx, b.eventsHandler, infraEnvID)
```

Comment:
append the error

Reply:
@YuviGold do you wish to include the error which is stored in `err.Error()` in the event message ?
This will present the error in the UI (while it is also written to the log)

Reply:
I am fine with both approaches. The error should at least be exposed somehow.

Reply:
it is logged with error severity in line 856 and the event message remains unchanged "Failed to download image: error fetching from storage backend" then this is covered here.